python.rst

```diff
@@ -822,6 +822,30 @@ Sources
   generated code and templates), but do not use globs as a shortcut to
   include files as sources.
 
+FLAG Guidelines
+===============
+
+Flag rules are related to core.flag library.
+
+Flag Naming
+-----------
+
+- Flag names should be lowercase (except in the case of secret values).
+- Don’t use stuttering names, flags are namespaced.
+   - For example: use timeline.page_size rather than timeline.timeline_page_size.
```

Comment:
Use double backtick to code format:
``timeline.page_size`` rather than ``timeline.timeline_page_size``.